### DISCUSSION TITLE
Editer des couches

### DISCUSSION THREAD

**Discussion by mangeniol** [mviewer/mviewer#857] [Q&A] [answered]
Bonjour,

Je voulais savoir s'il est possible d'éditer une couche de données géographiques avec Mviewer?

Merci d'avance pour vos réponses.

**Comment by Gaetanbrl** [comment]
Bonjour,
Mviewer est une visionneuse orientée vers la consultation des données.
Aucune fonctionnalité ne permet actuellement de réaliser de l'édition. Aucune évolution n'est prévue pour éditer des données.
D'autres outils permettent d'éditer des données tels que Qgis / MapStore..... Ce n'est la vocation du mviewer.

N'hésitez-pas si vous avez d'autres questions :)

**Reply by mangeniol** [reply, marked as answer]
Merci pour votre réponse rapide.
